apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ShareListView.kt

```diff
@@ -65,6 +65,16 @@ fun ShareListView(chatModel: ChatModel, stopped: Boolean) {
           hasSimplexLink = hasSimplexLink(mc.text)
         }
       }
+      is SharedContent.BulkForward -> {
+        sharedContent.chatItems.forEach {
```

Comment:
this seems to be wrong - it should be doing any (or w/e it's called in kotlin) instead of forEach - right now if you select multiple items, preferences are only checked against last item